<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="35230" started-at="2020-06-30T08:46:30 EDT" finished-at="2020-06-30T08:47:06 EDT">
    <groups>
    </groups>
    <test name="Test" duration-ms="35230" started-at="2020-06-30T08:46:30 EDT" finished-at="2020-06-30T08:47:06 EDT">
      <class name="com.qa.testcases.LoginTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.qa.testcases.LoginTest@528931cf]" name="setup" is-config="true" duration-ms="11163" started-at="2020-06-30T08:46:32 EDT" finished-at="2020-06-30T08:46:43 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="verifyLogin(java.lang.String, java.lang.String)[pri:0, instance:com.qa.testcases.LoginTest@528931cf]" name="verifyLogin" duration-ms="7062" started-at="2020-06-30T08:46:43 EDT" data-provider="getData" finished-at="2020-06-30T08:46:50 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[tutorial]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[tutorial]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogin -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.testcases.LoginTest@528931cf]" name="tearDown" is-config="true" duration-ms="317" started-at="2020-06-30T08:46:50 EDT" finished-at="2020-06-30T08:46:51 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.qa.testcases.LoginTest@528931cf]" name="setup" is-config="true" duration-ms="7962" started-at="2020-06-30T08:46:51 EDT" finished-at="2020-06-30T08:46:59 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="verifyLogin(java.lang.String, java.lang.String)[pri:0, instance:com.qa.testcases.LoginTest@528931cf]" name="verifyLogin" duration-ms="2688" started-at="2020-06-30T08:46:59 EDT" data-provider="getData" finished-at="2020-06-30T08:47:01 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[abcd]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[xyz]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Title does not match expected [Find a Flight: Mercury Tours:] but found [Sign-on: Mercury Tours]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Title does not match expected [Find a Flight: Mercury Tours:] but found [Sign-on: Mercury Tours]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at com.qa.testcases.LoginTest.verifyLogin(LoginTest.java:46)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogin -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.testcases.LoginTest@528931cf]" name="tearDown" is-config="true" duration-ms="4158" started-at="2020-06-30T08:47:01 EDT" finished-at="2020-06-30T08:47:06 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.testcases.LoginTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
